gemfirexd/client/src/main/java/com/pivotal/gemfirexd/internal/client/net/NetConnection.java

```diff
@@ -1341,7 +1341,7 @@ void rdbAccessed(int svrcod,
     public java.sql.CallableStatement getBucketToServerDetails_;
 
     public static final String BUCKET_AND_SERVER_PROC_QUERY =
-      "call SYS.GET_BUCKET_TO_SERVER_MAPPING(?, ?)";
+      "call SYS.GET_BUCKET_TO_SERVER_MAPPING2(?, ?)";
```

Comment:
So previous one which works with VARCHAR has been retained. and that is why 2 in the name. Can we not remove it or you want to keep it for if any user must be using?